tests/integration/targets/gitlab_group_members/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+# Test code for gitlab_group_members module
```

Comment:
Please add a file `tests/integration/targets/gitlab_group_members/aliases` with `unsupported` in it.